<ui version="4.0" >
 <class>QicsPrintDialogClass</class>
 <widget class="QDialog" name="QicsPrintDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QicsPrintDialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_5" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Page settings</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QPushButton" name="bPageSetup" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Setup</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QComboBox" name="cbPageFormat" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy" >
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Orientation:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbPageOrientation" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>Portrait</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Landscape</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Printer</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Printer Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lPrinterName" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bPrinterSetup" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Setup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bPrint" >
       <property name="text" >
        <string>Print</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Copies</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Order:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Number:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="sbCopies" >
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="cbCopiesOrder" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy" >
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text" >
          <string>Every Page (1,1,2,2,3,3...)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Every Set (1,2,3,...,1,2,3...)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="1" column="1" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Print order</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Pages:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Order:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="cbPageOrder" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy" >
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text" >
          <string>First to Last (1,2,3...)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Last to First (...3,2,1)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbPageSequence" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>All (1,2,3,4...)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Odd only (1,3,5...)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Even only (2,4,6...)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="2" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Pages to print</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="rbEnabledPages" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Enabled only</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="rbSelectedPages" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Selected only</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="rbAllPages" >
        <property name="text" >
         <string>All</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="rbCurrentPage" >
        <property name="text" >
         <string>Current</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="rbCustomPages" >
        <property name="text" >
         <string>Custom numbers:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="leCustomPages" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>bPrinterSetup</tabstop>
  <tabstop>cbPageOrientation</tabstop>
  <tabstop>cbPageFormat</tabstop>
  <tabstop>cbPageSequence</tabstop>
  <tabstop>cbPageOrder</tabstop>
  <tabstop>rbAllPages</tabstop>
  <tabstop>rbEnabledPages</tabstop>
  <tabstop>rbCurrentPage</tabstop>
  <tabstop>rbSelectedPages</tabstop>
  <tabstop>rbCustomPages</tabstop>
  <tabstop>leCustomPages</tabstop>
  <tabstop>sbCopies</tabstop>
  <tabstop>cbCopiesOrder</tabstop>
  <tabstop>bPrint</tabstop>
  <tabstop>bCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>QicsPrintDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>520</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel" >
     <x>326</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
